timesys/timing-devices-connection.txt

```diff
@@ -521,7 +521,7 @@ IA-12RaPS02:PS-UDC-SI1:SYNCOUT      IA-12RaPS02:PS-UDC-SI2:SYNCIN
 IA-12RaPS02:PS-UDC-SI4:SYNCOUT      IA-12RaPS02:PS-UDC-SI5:SYNCIN
 IA-13RaPS02:PS-UDC-SI1:SYNCOUT      IA-13RaPS02:PS-UDC-SI2:SYNCIN
 IA-13RaPS02:PS-UDC-SI4:SYNCOUT      IA-13RaPS02:PS-UDC-SI5:SYNCIN
-IA-14RaPS01:PS-UDC-SI4:SYNCOUT      IA-14RaPS01:PS-UDC-SI6:SYNCIN
+# IA-14RaPS01:PS-UDC-SI4:SYNCOUT      IA-14RaPS01:PS-UDC-SI6:SYNCIN
```

Comment:
remove the line instead of commenting it.

Reply:
this comment is just a remeber to generate thsi file properly, with the script.